File: JavaScript/0283_move_zeros/solution.js
function move_zeros(nums) {
    let replica = [];
    for (let num of nums) {
        if (num != 0) {
            replica.push(num);
        }
    }
    const count = nums.length - replica.length;
    for (let i = 1; i <= count; i ++) {
        replica.push(0)
    }
    for (let i = 0; i < nums.length; i ++) {
        nums[i] = replica[i];
    }
    return nums;
}

const nums = [0,1,0,3,12];
console.log(move_zeros(nums));

File: Python/0283_move_zeros/solution.py
def move_zeros(nums):
    replica = list()
    for num in nums:
        if num != 0:
            replica.append(num)
    for _ in range(len(nums) - len(replica)):
        replica.append(0)    
    for i in range(len(replica)):
        nums[i] = replica[i]
        
    return nums

nums = [0,1,0,3,12]
print(move_zeros(nums))